tests/test_md.py

```diff
@@ -917,14 +926,11 @@ def test_context_invalid_boxes():
     ctxt.set_box(box * 0.01)
     err_msg = "cutoff with padding is more than half of the box width, neighborlist is no longer reliable"
     with pytest.raises(RuntimeError, match=err_msg):
-        _, boxes = ctxt.multiple_steps(steps)
-        assert len(boxes) == 1
```

Comment:
Guessing these were removed because multiple_steps now returns void - any way to test number of actual steps taken/frames generated?

Reply:
These were removed because `multiple_steps` raises an exception, which means the checks for boxes never runs. Just removes dead code. 

We do test the number of steps taken and the frames generated in : https://github.com/proteneer/timemachine/blob/task/avoid-duplicate-memory-alloc/tests/test_md.py#L222-L241